torch/csrc/cuda/Module.cpp

```diff
@@ -1183,6 +1184,9 @@ static PyObject* THCPModule_initExtension(PyObject* self, PyObject* noargs) {
   set_module_attr("default_generators", default_cuda_generators);
   bindGetDeviceProperties(m);
 
+  // initialize kineto to fix #75504
+  torch::profiler::impl::kineto::initializeKineto();
```

Comment:
cc @ngimel - is this the right place to put this in order to initialize kineto during lazy cuda initialization?

Reply:
I'm not entirely sure - it might initialize context on the GPU before it needs to be initialized, which we don't want to happen.